src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java

```diff
@@ -7,14 +7,18 @@
 public class FlowersUnderSnowInBouquetChecker {
     public static Flower checkFlower(Flower flower) {
         if (flower instanceof SpringFlower) {
-            SpringFlower springFlower = (SpringFlower) flower;
-            if (springFlower.hasSeem()) {
-                return springFlower;
-            } else {
-                return GeneralStoreFlowerRandomizer.generateRandomStoreFlower();
-            }
+            return flowersSubstitute(flower);
         } else {
             return flower;
         }
     }
+
+    private static Flower flowersSubstitute(Flower flower) {
```

Comment:
A name of a method should be started from the verb.

Reply:
Done